### PR TITLE
Update deps

### PR DESCRIPTION
**Pull request by DeathByTape**
First pass at updating dependencies. Some thing we should explore further is the integration test.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -126,6 +126,12 @@
                 <scope>test</scope>
             </dependency>
             <dependency>
+                <groupId>org.glassfish.hk2</groupId>
+                <artifactId>hk2-utils</artifactId>
+                <version>2.4.0-b32</version>
+                <scope>test</scope>
+            </dependency>
```
Comment (danchen) [inline pom.xml]: Why do we need hk2 here?

Comment (DeathByTape) [inline pom.xml]: When I upgraded, this was missing. I believe rest-assured uses it in our integration tests.

File: pom.xml [hunk 2]
```diff
@@ -221,7 +227,7 @@
                 </plugin>
                 <plugin>
                     <artifactId>maven-release-plugin</artifactId>
-                    <version>2.5.2</version>
+                    <version>3.0-r1585899</version>
```
Comment (danchen) [inline pom.xml]: I think we should use: 2.5.3 (searching from search.maven.org)

File: pom.xml (hunk 1 above)
Comment (danchen) [inline pom.xml]: Ok, I think we can push this up into elide-integration-test, and remove it from here. Just, tried it and I can make this commit.

File: pom.xml (hunk 2 above)
Comment (DeathByTape) [inline pom.xml]: Why?

File: pom.xml (hunk 1 above)
Comment (DeathByTape) [inline pom.xml]: Tracked it down.

**origin/master**
```
[INFO] com.yahoo.elide:elide-integration-tests:jar:1.0.0.4-SNAPSHOT
[INFO] +- com.yahoo.elide:elide-core:jar:1.0.0.4-SNAPSHOT:compile
[INFO] |  +- com.google.guava:guava:jar:16.0.1:compile
[INFO] |  +- org.glassfish.hk2:hk2-api:jar:2.4.0-b25:compile
[INFO] |  |  +- org.glassfish.hk2:hk2-utils:jar:2.4.0-b25:compile
[INFO] |  |  \- org.glassfish.hk2.external:aopalliance-repackaged:jar:2.4.0-b25:compile
[INFO] |  +- org.antlr:antlr4-runtime:jar:4.5:compile
```

**updateDeps** branch
```
[INFO] com.yahoo.elide:elide-integration-tests:jar:1.0.0.4-SNAPSHOT
[INFO] +- com.yahoo.elide:elide-core:jar:1.0.0.4-SNAPSHOT:compile
[INFO] |  +- com.google.guava:guava:jar:18.0:compile
[INFO] |  +- org.glassfish.hk2:hk2-api:jar:2.4.0-b25:compile
[INFO] |  |  \- org.glassfish.hk2.external:aopalliance-repackaged:jar:2.4.0-b25:compile
[INFO] |  +- org.antlr:antlr4-runtime:jar:4.5.1-1:compile
```

Not entirely sure what changed here. In any case, it appears to be fixed by bumping the hk2 version (see updated commit.)

**Comment by danchen** [comment]
:+1:

**Comment by yahoocla** [comment]
CLA is valid!

**Pull request merged**